packages/ui/src/components/TextArea/index.tsx

```diff
@@ -0,0 +1,288 @@
+  placeholder?: string
+  /**
+   * Override others properties : readyOnly, success, error.
+   */
+  disabled?: boolean
+  /**
+   * Override others properties : success, error.
+   * Ignored if following props are provided : disabled.
+   */
+  readOnly?: boolean
+  /**
+   * Override others properties : error, helper.
+   * Ignored if following props are provided : disabled, readyOnly.
+   */
+  success?: string
+  /**
+   * Override others properties : helper.
+   * Ignored if following props are provided : disabled, readyOnly, success.
+   */
+  error?: string
+  /**
+   * Ignored if following props are provided : readyOnly, success.
+   */
+  helper?: ReactNode
+  rows?: number
+  minLength?: number
+  maxLength?: number
+  tooltip?: string
+  required?: boolean
+  'data-testid'?: string
+  name?: string
+  onFocus?: DOMAttributes<HTMLTextAreaElement>['onFocus']
+  onBlur?: DOMAttributes<HTMLTextAreaElement>['onBlur']
+  clearable?: boolean
+  labelDescription?: ReactNode
+}
+
+/**
+ * Descriptionnnnnn
+ */
+export const TextArea = forwardRef<HTMLTextAreaElement, TextAreaProps>(
+  (
+    {
+      id,
+      value,
+      onChange,
+      placeholder,
+      rows = 3,
+      disabled = false,
+      readOnly = false,
+      success,
+      error,
+      helper,
+      minLength,
+      maxLength,
+      tooltip,
+      label,
+      autoFocus,
+      required = false,
+      'data-testid': dataTestId,
+      name,
+      onFocus,
+      onBlur,
+      clearable = false,
+      labelDescription,
+    },
+    ref,
+  ) => {
+    const localId = useId()
+
+    // Avoid conflicts between properties
+    const computedReadOnly = !disabled && readOnly
+    const computedSuccess = !disabled && !readOnly ? success : undefined
+    const computedError = !disabled && !readOnly && !success ? error : undefined
+    const computedHelper = !success && !error ? helper : undefined
+
+    return (
+      <Stack gap={1}>
+        <Stack direction="row" gap="1" alignItems="center">
+          <Stack direction="row" gap="0.5" alignItems="start">
+            <Text
+              as="label"
+              variant="bodySmallStrong"
+              sentiment="neutral"
+              htmlFor={id ?? localId}
+            >
+              {label}
+            </Text>
+            {required ? <Icon name="asterisk" color="danger" size={8} /> : null}
+          </Stack>
+          {labelDescription ?? null}
+        </Stack>
+        <div>
+          <Tooltip text={tooltip}>
+            <StyledTextAreaWrapper>
+              <StyledTextArea
+                id={id ?? localId}
+                autoFocus={autoFocus}
+                disabled={disabled}
+                rows={rows}
+                ref={ref}
+                value={value}
+                onChange={event => {
+                  onChange(event.currentTarget.value)
+                }}
+                isSuccess={!!computedSuccess}
+                isError={!!computedError}
```

Comment:
I just read that `aria-invalid` can be a good addition but not required, so maybe we should add it too

> When visible text is used to programmatically identify a failed field and / or convey how the error can be corrected, setting aria-invalid to "true" is not required from a strict compliance standpoint but may still provide helpful information for users.

Source: https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA21.html#:~:text=When%20visible%20text%20is%20used,provide%20helpful%20information%20for%20users.